ONSdigital/eq-author
# eq-380 Author S3 Integration

### Changes

- Replaces node backend with Python app and Amazon S3 integration as per #8 
- Re-integrate webpack bundles with django templates

### How to test

- ensure local database is setup correctly before testing (ie. the API is working)
- `npm install`
- `npm start`
- run the python app `./scripts/run_app.sh`
- visit `http://localhost:8000`

### Who can test

- anyone
